psi4/src/psi4/libfock/points.cc

```diff
@@ -702,6 +702,10 @@ void BasisFunctions::compute_functions(std::shared_ptr<BlockOPoints> block) {
     double* x = block->x();
     double* y = block->y();
     double* z = block->z();
+    double* xyz = new double[npoints * 3];
+    ::memcpy(xyz, x, sizeof(double) * npoints);
+    ::memcpy(xyz + npoints, y, sizeof(double) * npoints);
+    ::memcpy(xyz + 2 * npoints, z, sizeof(double) * npoints);
```

Comment:
The C++ way to do this is with either `std::copy` or `std::copy_n`.  Let's leave this as-is for now and I'll update it when i reconcile with the GGA hessian changes

Reply:
Right, I couldn't find that branch you mentioned. I've had my learning exercise -- glad to see the modern way.